ts/src/connections/tunnelClientBase.ts

```diff
@@ -29,6 +33,10 @@ import { Emitter } from 'vscode-jsonrpc';
 import { TunnelConnectionSession } from './tunnelConnectionSession';
 import { TunnelManagementClient } from '@microsoft/dev-tunnels-management';
 import { tunnelSshSessionClass } from './tunnelSshSessionClass';
+import { PortRelayConnectResponseMessage } from './messages/portRelayConnectResponseMessage';
+
+export const webSocketSubProtocol = 'tunnel-relay-client';
```

Comment:
Nit: can we use `clientWebSocketSubProtocol` and `hostWebSocketSubProtocol` constant names? Otherwise they may get confused which is which.

Reply:
The _public_ constants are distinct because they are properties of different classes. These constants are private; they are exported only for use by the base class, but are not public because they are not exported by `index.ts`.